src/keychain.js

```diff
@@ -175,223 +172,195 @@ class Keychain {
    * @param {string} name - The local key name; cannot already exist.
    * @param {string} type - One of the key types; 'rsa'.
    * @param {int} size - The key size in bits.
-   * @param {function(Error, KeyInfo)} callback
    * @returns {undefined}
```

Comment:
Should we also specify in jsdocs the content of `KeyInfo`?

```suggestion
    * @returns {KeyInfo}
```